changelog

```diff
@@ -1,3 +1,202 @@
+      * Remove outdated watchdog. (@petervdmeer)
+      * Properly check if a mission is failed. (@TomGoodIdea)
+      * Hardpoint labels and lines in the ship info panel will properly align to the "weapons" box from the interface definition. (@Zitchas)
+  * Game content:
+    * New content:
+      * Added a mission transporting a Hai to Unfettered space. (@roadrunner56)
+      * Added Unfettered-related Hai news. (@Hurleveur)
+      * Added missions about Hogshead referencing Boaty McBoatface story. (@Saugia)
+      * Added Coalition culture conversations. (@Arachi-Lover)
+      * Added new human merchant hails about mining asteroids. (@bene_dictator)
+      * Create more passenger jobs requiring keystones in Hai space. (@roadrunner56)
+      * Add more human news referring to Korath attacks in the Core. (@roadrunner56)
+      * Added conspiracy theorist spaceport news. (@tibetiroka)
+      * Added logbook entries for Alastair Korban. (@bene_dictator)
+      * Added a conversation about Kraz and Delta V outfits being stocked in the northern Dirt Belt and near Earth after the war begins. (@bene_dictator)
+      * Added a short set of missions involving the Wanderers and Mereti. (@Quantumshark)
+      * Updated Kraken Station landing image. (@bene_dictator)
+      * Added new planets and stations to Human space. (@bene_dictator, @Quantumshark)
+      * Added normal blaster weapons to sales on pirate planets. (@bene_dictator)
+      * Added missions about bringing a lost dog to its owner on Glory. (@SpearDane, @bene_dictator)
+      * Added Core Pirate ship variants with Tractor Beams. (@Saugia)
+      * Added derelict ships that can randomly spawn in some human systems. (@Petersupes)
+      * Added a series of missions about a journalist in Syndicate space. (@bene_dictator)
+    * Mission changes:
+      * Refactor the Argosy Hijacking mission. (@bene_dictator)
+      * Reduce the requirements for unlocking the Kestrel the first time. (@brendanjones)
+      * Make the "Remnant: Cognizance 4" mission description more helpful. (@OcelotWalrus)
+      * Various improvements to pre-war missions. (@Anarchist2)
+        * Text changes to all chains.
+        * Care Package to South:
+          * Limit the source to near earth planets, increase offer rate to 65%.
+          * Reworked the way the follow-up missions are handled:
+          * Care Package 2a has been replaced with a new mission, FW Deep Memorial, a set of missions and events that offers regardless of the player's choices, but which has a branch if Brower dies during the attack.
+          * Care Package 2b has been moved from immediately after Diplomacy 4 to during the first FW break, and now offers on Rust instead of Deep. The flamethrower mission will likely bring players to Rust anyway, but it's no longer in the way of the main campaign.
+        * Moving House:
+          * Increase offer rate to 35%, limit source planets to near earth, core and paradise.
```

Comment:
```suggestion
          * Increase offer rate to 35%, limit source planets to near Earth, core and paradise.
```

Reply:
```suggestion
          * Increase offer rate to 35%, limit source planets to "near earth", "core" and "paradise".
```
Alternatively, the other two should be capitalised since they are proper names of those regions.
```suggestion
          * Increase offer rate to 35%, limit source planets to near Earth, Core and Paradise.
```

Reply:
The first one is probably better. You might have to change some I didn't comment on, too.